src/components/[guild]/EditGuild/components/TagManager.tsx

```diff
@@ -0,0 +1,39 @@
+import { HStack } from "@chakra-ui/react"
+import Switch from "components/common/Switch"
+import { useEffect } from "react"
+import { useForm } from "react-hook-form"
+import { GuildTags } from "types"
+
+type Props = {
+  tags: GuildTags[]
+  setTags: (tags: GuildTags[]) => void
+}
+
+const TagManager = ({ tags, setTags }: Props): JSX.Element => {
+  const { register, watch } = useForm<{ [k in GuildTags]: boolean }>({
+    defaultValues: {
+      VERIFIED: tags.includes("VERIFIED"),
+      FEATURED: tags.includes("FEATURED"),
+    },
+  })
+
+  useEffect(() => {
```

Comment:
I think we could just use [useFormContext](https://react-hook-form.com/docs/useformcontext) here and put these tags directly into the edit guild form (so we wouldn't need this logic & the tags/setTags logic), then handle the API calls in handleSubmit (or, in other words, we could just put the post/delete requests from `useEditTags` to `useEditGuild`).

Also, it's pretty strange for me that we have separate endpoints just for the tags, but I guess that was a core decision?

Reply:
The Core decision was that use a separate tags endpoint to tend to RESTful. In the first version, I used the guild endpoint to edit the tags along with the other properties.

However, based on your suggestion, I'll try to make some changes here.

Reply:
All right, I think we could send these tag-related requests in `useEditGuild`, just like we did [in `useEditRole`](https://github.com/agoraxyz/guild.xyz/blob/main/src/components/%5Bguild%5D/RoleCard/components/EditRole/hooks/useEditRole.ts#L96-L108). What do you think?